formats/cyclonedx/cyclonedx_test.go

```diff
@@ -18,3 +18,14 @@ func TestPurls(t *testing.T) {
 	assert.Len(t, purls, 1)
 	assert.Equal(t, "pkg:golang/github.com/CycloneDX/cyclonedx-go@v0.6.0", purls[0])
 }
+
+func TestLicenses(t *testing.T) {
+	var sbom cyclone.BOM
```

Comment:
Should line 23 be cyclonedx.BOM instead of "cyclone.BOM"?

Reply:
It's actually a reference back to ```cyclone "github.com/CycloneDX/cyclonedx-go```